File: fun_fortran/mySqrtNew.f90
!!SQRT Riemann Sheets
subroutine mySqrtNew(z,w)
implicit none
complex*16 z,w
real*8 tmp1, tmp2
!
w=sqrt(z)
if  (imag(w) > 0.d0) then
    tmp1=real(w)
    tmp2=-imag(w)
    w=cmplx(tmp1,tmp2)
end if
if (real(w) < 0.d0) then
    tmp1=-real(w)
    tmp2=imag(w)
    w=cmplx(tmp1,tmp2)
end if
end subroutine mySqrtNew

File: fun_matlab/mySqrtNew.m
%% SQRT Riemann Sheets
function[w]=mySqrtNew(z)
% w       =   sqrt(z);
% w       =   w.*(imag(w)<=0)-w.*(imag(w)>0); % Proper Sheet
w=sqrt(z);
for ii = 1:length(z)
if imag(w(ii))>0
    w(ii)=real(w(ii))-1j*imag(w(ii));
end
if real(w(ii))<0
   w(ii)=-real(w(ii))+1j*imag(w(ii)); 
end
end
end